win32ss/gdi/gdi32/objects/bitmap.c

```diff
@@ -767,14 +742,12 @@ SetDIBitsToDevice(
      pConvertedInfo->bmiHeader.biCompression  == BI_PNG )) )*/
     {
         LinesCopied = NtGdiSetDIBitsToDeviceInternal(hdc, XDest, YDest, Width, Height, XSrc, YSrc,
-            StartScan, ScanLines, (LPBYTE) pvSafeBits, (LPBITMAPINFO) pConvertedInfo, ColorUse,
+            StartScan, ScanLines, (LPBYTE) Bits, (LPBITMAPINFO) pConvertedInfo, ColorUse,
```

Comment:
Note: the cast in front of `pConvertedInfo` shouldn't be necessary, as it is already of the correct type.